src/scipp/html/formatting_datagroup_html.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+# Original source from
+# https://github.com/jsignell/xarray/blob/1d960933ab252e0f79f7e050e6c9261d55568057/xarray/core/formatting_html.py
```

Comment:
I think this comment can be removed, since it was about the code for the `DataArray` repr.